internal/forms/forms.go

```diff
@@ -1010,6 +1018,8 @@ func (b formMessageBuilder) initFormValues() {
 }
 
 func (b formMessageBuilder) scanTmplBuildMessage(tmplPath string) (MessageForm, error) {
+	var inBody = false
```

Comment:
I think this should be moved to just above the loop, to make it easier to understand what it relates to. Also, declaring a bool like this is very unusual in Go. Consider writing it at `inBody := false` or `var inBody bool`.